plugins/woocommerce/src/StoreApi/Routes/V1/Checkout.php

```diff
@@ -224,6 +263,54 @@ public function validate_required_additional_fields( \WP_REST_Request $request )
 		}
 	}
 
+	/**
+	 * Get route response for PUT requests.
+	 *
+	 * @param \WP_REST_Request $request Request object.
+	 * @throws RouteException On error.
+	 * @return \WP_REST_Response
+	 */
+	protected function get_route_update_response( \WP_REST_Request $request ) {
+		/**
+		 * Create (or update) Draft Order and process request data.
+		 */
+		$this->create_or_update_draft_order( $request );
+
+		/**
+		 * Persist additional fields, order notes and payment method for order.
+		 */
+		$this->persist_additional_fields_for_order( $request );
+		$this->persist_order_notes_for_order( $request );
+		$this->persist_payment_method_for_order( $request );
+
+		if ( $request->get_param( 'calc_totals' ) ) {
```

Comment:
Possibly needs experimental prefix while in flux.

Reply:
Done in [Add experimental prefix to calc_totals](https://github.com/woocommerce/woocommerce/pull/53553/commits/a1a581ac26e2067acb1315eadcf85afea5ed72f9)